src/component/left-pane.js

```diff
@@ -39,15 +38,15 @@ class LeftPane extends Component {
 							</button>
 						</div>
 					</div>
-            
+
 					<List className="user-list">
 						{this.state.userList.map((user, index) => {
 							const className = user === this.props.currentSelected ? 'active' : null;
 							return (
 								<div  key={index} onClick={() => (this.props.userSelected(user))}>
 									<ListItem dense button className={className}>
 										<Avatar><Icon>person</Icon></Avatar>
-										<ListItemText  primary={user} />
+										<ListItemText primary={user} className="username" />
```

Comment:
Guess this will cause a UI change. Can I see before/after screenshot for this?

Reply:
> Guess this will cause a UI change. Can I see before/after screenshot for this?


![Desktop 2019-07-08 00-13-01](https://user-images.githubusercontent.com/11846339/60772509-3067b980-a115-11e9-93eb-ed7bebf51442.png)